src/cmclient/cmclient.cpp

```diff
@@ -118,6 +120,76 @@ aos::Error CMClient::SendImageContentRequest(const ImageContentRequest& request)
     return aos::ErrorEnum::eNone;
 }
 
+aos::Error CMClient::SendMonitoringData(const aos::monitoring::NodeMonitoringData& monitoringData)
+{
+    aos::LockGuard lock(mMutex);
+
+    LOG_DBG() << "Send monitoring data";
+
+    mOutgoingMessage.which_SMOutgoingMessage = servicemanager_v3_SMOutgoingMessages_node_monitoring_tag;
+
+    mOutgoingMessage.SMOutgoingMessage.node_monitoring
+        = servicemanager_v3_NodeMonitoring servicemanager_v3_NodeMonitoring_init_zero;
+
+    mOutgoingMessage.SMOutgoingMessage.node_monitoring.has_monitoring_data = true;
+
+    mOutgoingMessage.SMOutgoingMessage.node_monitoring.monitoring_data.cpu = monitoringData.mMonitoringData.mCPU;
+    mOutgoingMessage.SMOutgoingMessage.node_monitoring.monitoring_data.ram = monitoringData.mMonitoringData.mRAM;
+
+    mOutgoingMessage.SMOutgoingMessage.node_monitoring.monitoring_data.disk_count
+        = monitoringData.mMonitoringData.mDisk.Size();
+    for (size_t i = 0; i < monitoringData.mMonitoringData.mDisk.Size(); i++) {
+        mOutgoingMessage.SMOutgoingMessage.node_monitoring.monitoring_data.disk[i]
+            = PartitionInfoToPB(monitoringData.mMonitoringData.mDisk[i]);
+    }
+
+    mOutgoingMessage.SMOutgoingMessage.node_monitoring.instance_monitoring_count
+        = monitoringData.mServiceInstances.Size();
+    for (size_t i = 0; i < monitoringData.mServiceInstances.Size(); i++) {
+        mOutgoingMessage.SMOutgoingMessage.node_monitoring.instance_monitoring[i]
+            = InstanceMonitoringToPB(monitoringData.mServiceInstances[i]);
+    }
+
+    auto current_time = time(NULL);
+
+    struct timespec spec;
```

Comment:
`NodeMonitoringData` has timestamp field and should be filled by monitoring module rather than in cmclient.

Reply:
Done